sc.it.homework.Concurrency/src/main/java/Concurrency/FestivalGate.java

```diff
@@ -0,0 +1,32 @@
+package Concurrency;
+import java.util.LinkedList;
+public class FestivalGate {

```

Comment:
Gate shouldn't have the main, main should be in a starting class, Gate is separate